src/ImageTracking.jl

```diff
@@ -59,6 +59,7 @@ export
 
     # tracking algorithms
     BoxROI,
-    TrackerBoosting
+    TrackerBoosting,
+    init_tracker
```

Comment:
Sorry, my bad. I missed update_tracker. 


> How about exporting a wrapper to `init_tracker` and `update_tracker`? It can be something like:

I don't think we should do that. From a user point of view, I will like to first `initialize` the tracker and then `update` it clearly and not by using a wrapper, write same things for `init` and `update`. But I can add that, as it won't affect anything?